Makefile

```diff
@@ -27,7 +27,7 @@ export CARGO_BIN_DIR ?= $(shell echo "${HOME}/.cargo/bin")
 export GIT_COMMIT ?= $(shell git rev-parse --short HEAD)
 
 # Specific versions of various Rust tools we use.
-export CARGO_TOOL_VERSION_dd-rust-license_tool ?= 1.0.3
+export CARGO_TOOL_VERSION_dd-rust-license-tool ?= 1.0.3
```

Comment:
The name structure was wrong so if `dd-rust-license-tool` wasn't already present, the fallback code in the Makefile to install it would fail to run.